src/Files.App/Services/PreviewPopupProviders/QuickLookProvider.cs

```diff
@@ -16,6 +17,7 @@ public sealed class QuickLookProvider : IPreviewPopupProvider
 		private static string pipeName = $"QuickLook.App.Pipe.{WindowsIdentity.GetCurrent().User?.Value}";
 		private static string pipeMessageSwitch = "QuickLook.App.PipeMessages.Switch";
 		private static string pipeMessageToggle = "QuickLook.App.PipeMessages.Toggle";
+		private static Encoding encoding = Encoding.GetEncoding("UTF-8", new EncoderReplacementFallback("?"), new DecoderExceptionFallback());
```

Comment:
What's the reason put this here? This should be instantiate right before using and GC'd right after the method finished.